docs/source/deployment/deploy-model-locally.rst

```diff
@@ -77,7 +77,59 @@ CSV input must be a valid pandas.DataFrame CSV representation. For example:
 JSON Input
 **********
 
-JSON input must be a dictionary with exactly one of the following fields, further specifying the input data's type and encoding:
+You can either pass a flat dictionary corresponding to the desired model payload or wrap the
+payload in a dict with a dict key that specifies your payload format. 
+
+Raw Payload Dict
+^^^^^^^^^^^^^^^^
+
+If your payload is in a format that your mlflow served model will accept and it's in the supported
+models below, you can pass a raw payload dict.
+
+.. list-table::
+    :widths: 20 40 40
+    :header-rows: 1
+    :class: wrap-table
+
+    * - Supported Model Type
+      - Description
+      - Example
+    * - OpenAI Chat
+      - `OpenAI chat request payload <https://platform.openai.com/docs/api-reference/chat/create>`_.†
+      - 
+        .. code-block:: python
+
+          {"messages": [{"role": "user", "content": "Tell a joke!"}], "temperature": 0.0}
```

Comment:
nit: could you format this with indentation? (Table view is not truncated so has relatively low durability for line length)
```
{
    "messages": [
        {
            "role": "user",
            "content": ...
```

Reply:
Linting reformats. I noqa-ed, but for ruff I don't think there's a convenient noqa range operator. Instead of commenting `# noqa` on each line, I reformatted to the current commit. Lmk if I should add more to better align with your comment

Reply:
Rendered docs fit on my browser now

Reply:
yea separating params is good enough, thx!